gemspec.rb

```diff
@@ -3,12 +3,13 @@ def specification(version, default_adapter, platform = nil)
     s.specification_version = 2 if s.respond_to? :specification_version=
     s.required_rubygems_version = Gem::Requirement.new('>= 0') if s.respond_to? :required_rubygems_version=
     s.rubygems_version = '0.0.1'
-    s.required_ruby_version = '>= 1.9'
+    s.required_ruby_version = '>= 2.1'
 
     s.name              = 'gollum-lib'
     s.version           = version
     s.platform          = platform if platform
-    s.date              = '2018-10-01'
+    s.date              = '2020-03-12'
```

Comment:
To update.

Reply:
The `rake build` task will take care of this 👍